tests/x11/thunderbird/thunderbird_imap.pm

```diff
@@ -39,15 +40,16 @@ sub run {
     }
 
     mouse_hide(1);
+
     # clean up and start thunderbird
     x11_start_program("xterm -e \"killall -9 thunderbird; find ~ -name *thunderbird | xargs rm -rf;\"", valid => 0);
-    my $success = eval { x11_start_program("thunderbird", match_timeout => 300); 1 };
+    my $success = eval { x11_start_program("thunderbird", match_timeout => 120); 1 };
```

Comment:
I don't see the reason for this change. I assume the rather high timeout was selected in the past based on observations where tests failed due to this. Why lower it?

Reply:
The high value was used on my lab,  because my openqa lab have poor resources to run. Correct is 120.